source/index.html.erb

```diff
@@ -596,37 +596,91 @@ per_page: 10
 
     <section class="resume-section p-3 p-lg-5 d-flex flex-column section-scroll" id="followus" data-nav-color="yellow">
       <div class="my-auto">
-        <div class="row justify-content-center">
-          <h2 class="mb-5" id="follow">
+        <div>
+          <h2 class="mb-5" id="follow"  align="center">
```

Comment:
Please use a CSS class instead of align attribute, .text-center should do the job